core.yaml

```diff
@@ -229,6 +357,14 @@ adminApps:
         namespace: kubeapps
         type: public
         # auth: true
+    info:
+      appVersion: 2.4.3
```

Comment:
We need to either automate appVersion change in this file or document in development guide that after upgrading a given chart appVersion needs to be changed here

Reply:
automation is gonna be messy, so we just have to maintain it